ToDoApp/ToDoApp/CreateTaskView.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  CreateTaskView.swift
+//  ToDoApp
+//
+//  Created by Trong Hieu Vuong on 2022-12-02.
+//
+
+import Foundation
+import SwiftUI
+
+struct CreateTaskView: View {
+    @StateObject private var createTaskViewModel = CreateTaskViewModel()
+    @State var titleInput: String = ""
+    @State var showDatePicker = false
```

Comment:
we will implement displaying date picker in a different way, will show you below. this may be removed